vignettes/execution.Rmd

```diff
@@ -0,0 +1,97 @@
+messages, output, and result.  This must be passed an executable R file to run and create the log elements from.
+
+* `log_write()` to generate and format the log
+
+* `log_remove()` to remove the log.rx environment created by code exeuction
+
+```{r log_*, eval = FALSE}
+log_config("my_script.R")
+run_safely_loudly("my_script.R")
+log_write()
+log_remove()
+```
+
+## Command Line Execution
+While exeucting from an R terminal is nice if you have access to one, you can also execute your code using system command line.  This is done using the `Rscript -e` command which executes a file using the registered Rscript executable.  Below are a few examples of how to use the command line to execute a file and create a log as well as how to manipulate the outputs of the execution.  These are likely to be advanced examples for most users.
+
+The below chunk will run the file my_script.R and output any standard output that is created by the execution of the file to the default location.
+```{r, engine = 'bash', eval = FALSE}
+Rscript -e "logrx::axecute('my_script.R', log_name = 'my_script.log', log_path = '.')"
+```
+
+The below chunk will run the file my_script.R and output any standard output that is created by the execution of the file to a file called temp.log.  This output can then later be accessed should the user want to check the output of the file execution.
+```{r, engine = 'bash', eval = FALSE}
+Rscript -e "logrx::axecute('my_script.R', log_name = 'my_script.log', log_path = '.')" > temp.log 2>&1
+```
+
+The below chunk will run the file my_script.R and output any standard output that is created by the execution of the file to a file called temp.log.  It will then take the contents of the temp.log file and add those to the end of the log file generated by the file execution and remove the temporary output log.
+```{r, engine = 'bash', eval = FALSE}
+Rscript -e "logrx::axecute('my_script.R', log_name = 'my_script.log', log_path = '.')" > temp.log 2>&1; cat my_script.log >> my_script.log; rm my_script.log
+```
+
+
+# Scripting with ```logrx```
+While ```logrx``` is built around creating a log for a program it can just as easily be used when running an entire set of programs.  The `axecute()` function has been built with both single file execution and scripted file execution in mind.  With the use of simple functions such as `lapply()` scripting is easy.  Below is some sample code of how `lapply()` can be used with `axecute()`.
+
+```{r scripting, eval = FALSE}
+lapply(c("file.R", "otherfile.R"), axecute)
+```
+
+If your scripting needs to work on the contents of a directory instead of a pre-defined list, functions such as `list.files` can be used to obtain a list of files to use.  Below is an example of how this can be applied in practice by getting all files ending in '.R' in the current working directory using a regular expression, and then using `lapply()` to run the files using `axecute()`.
+
```

Comment:
```suggestion
If your scripting needs to work on the contents of a directory instead of a pre-defined list, functions such as `list.files` can be used to obtain a list of files to use.  Below is an example of how this can be applied in practice by getting all files ending in `.R` in the current working directory using a regular expression, and then using `lapply()` to run the files using `axecute()`.

```